index.bs

```diff
@@ -630,7 +630,9 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 				1. Let |r| be the result of running [=check clipboard read permission=] [=in parallel=]
 
-				1. If |r| is not "granted", then reject |p| with a "NotAllowedError" DOMException
+				1. Let |f| be the result of checking clipboard feature policy [=in parallel=]
```

Comment:
I think the feature policy checks can be done sync. This should link to [=allowed to use=] instead saying "result of checking".

Reply:
Same for all the ones below...

Reply:
I'd put the following as step 1 (link as appropriate):


> If the current settings object's responsible document is not allowed to use the "clipboard-read" feature, then return a promise rejected with "NotAllowedError" DOMException.

Reply:
Thanks! I think I've addressed this comment, but please let me know if it's not placed in the right spot not.